index.html

```diff
@@ -167,14 +167,14 @@ <h2 class="heading-dark">Multi-Post Stories</h2>
       <section class="about">
         <div class="text">
           <h2 class="heading">About Myself</h2>
-    
+
           <p>
             I'm a software developer! I can help you build a product , feature or website Look
             through some of my work and experience! If you like what you see and have a project you
             need coded, don't hestiate to contact me.
           </p>
           <a href="#" class="let-connect">LET'S CONNECT</a>
-    
+
```

Comment:
You can connect all your social handles to it so as to take users to their respective links